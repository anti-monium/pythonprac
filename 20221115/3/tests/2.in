alp = Alpha(d=3, k=7, i=-2, v=10)
try:
	print(alp.a)
except AttributeError:
	print('there is no a')
try:
	print(alp.abc)
except AttributeError:
	print('there is no abc')
print(alp)
alp = AlphaQ(d=3, k=7, i=-2, v=10)
try:
	print(alp.a)
except AttributeError:
	print('there is no a')
try:
	print(alp.abc)
except AttributeError:
	print('there is no abc')
print(alp)
