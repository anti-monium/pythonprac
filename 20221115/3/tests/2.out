there is no a
there is no abc
d: 3, i: -2, k: 7, v: 10
there is no a
there is no abc
d: 3, i: -2, k: 7, v: 10
